src/lib/components/chart.svelte

```diff
@@ -2,7 +2,7 @@
 	import Chart, { type ChartDataset } from 'chart.js/auto'
 
 	interface Props {
-		labels: (number | string)[]
+		labels: number[]
```

Comment:
Lets keep it as it was before :)
```suggestion
		labels: (number | string)[]
```